dev-scripts/run_tests

```diff
@@ -2,6 +2,12 @@
 
 set -ex
```

Comment:
trivial: let's name this `run_tests.sh`

it's currently inconsistent where some shell scripts in this folder end in `.sh` and others don't